apps/expo/src/utils/timerFormatter.ts

```diff
@@ -0,0 +1,12 @@
+const formatTime = (time: number) => {
+    const hours = Math.floor(time / 3600)
+    const minutes = Math.floor(time / 60) % 60
+    const seconds = time % 60
+    return hours
```

Comment:
Could you please improve the readability on this? Nested ternaries are typically considered bad practice.